regcore_read/views/es_search.py

```diff
@@ -37,6 +39,10 @@ def search(request, doc_type):
             term['version'] = version
         if regulation:
             term['regulation'] = regulation
+        if is_root:
+            term['is_root'] = is_root
+        if is_subpart:
+            term['is_subpart'] = is_subpart
```

Comment:
Question about validating query parameters--do we prefer to throw an error on bad values, or silently fail? I'm asking because I see examples of each: comment submission errors on bad values, but search silently ignores invalid paging parameters.